fair_mango/dataset/dataset.py

```diff
@@ -36,19 +41,21 @@ def check_column_in_df(df: pd.DataFrame, columns: Sequence) -> None:
 def check_real_and_predicted_target_match(
     real_target: Sequence[str], predicted_target: Sequence[str]
 ) -> None:
-    """check that number of real targets and number of predicted targets match
+    """check that the number of real targets and number of predicted targets
+    match.
 
     Parameters
     ----------
     real_target : Sequence[str]
-        real targets
+        sequence of column names corresponding to the real targets
+        (true labels).
     predicted_target : Sequence[str]
-        predicted targets
+        sequence of column names corresponding to the predicted targets.
```

Comment:
question: how flexible are you regarding Numpy DocString convention ? I ask because their descriptions start with a capital and end with a dot.
https://numpydoc.readthedocs.io/en/latest/format.html#parameters

Reply:
Good question, I tend to use lowercase letters in the beginning of parameter descriptions. I will change that to use capital letters and I will try to be more consistent with the dot at the end.